src/Identity/Core/src/Passkeys/COSEAlgorithmIdentifier.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Identity;
+
+/// <summary>
+/// Represents a number identifying a cryptographic algorithm.
+/// </summary>
+/// <remarks>
+/// See <see href="https://www.w3.org/TR/webauthn-3/#typedefdef-cosealgorithmidentifier"/>.
+/// </remarks>
+internal enum COSEAlgorithmIdentifier : long
```

Comment:
```suggestion
internal enum COSEAlgorithmIdentifier : int
```

Should this be int? It looks like the Web IDL standard defines "long" to be 32 bits. https://webidl.spec.whatwg.org/#idl-long

Reply:
Wow yes, good catch!